doc/nrf/ug_thread_tools.rst

```diff
@@ -115,3 +142,120 @@ The most common wpanctl commands are the following:
   For example, ``set NCP:SleepyPollInterval 1000`` will set the value of the ``NCP:SleepyPollInterval`` property to ``1000``.
 
 For the full list of commands, run the ``help`` command in wpanctl.
+
+.. _ug_thread_tools_pyspinel:
+
+PySpinel
+********
+
+`PySpinel`_ is a tool for controlling OpenThread Network Co-Processor instances through a command-line interface.
```

Comment:
It's probably incorrect now that it's only for NCP - right? (I moved that line from the introduction.)

Reply:
Yes, PySpinel is a tool for NCP and RCP. In general for co-processor architecture.

Reply:
Updated.